### PR TITLE
Added message errors

### PR DESCRIPTION
**Pull request by KsquarianMike**
The deployed app doesn't show errors when cant create a booking

### CODE REVIEW

File: .nyc_output/79309f39-2d6f-41b8-8147-14837f520fa6.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (mishelashala) [inline .nyc_output/79309f39-2d6f-41b8-8147-14837f520fa6.json]: add `.nyc_output` to `.gitignore`

File: app/services/GoogleCalendarService.ts [hunk 2]
```diff
@@ -40,7 +40,9 @@ class GoogleCalendarService {
       .then(result => {
         return result;
       })
-      .catch(err => Controller.serverError(err));
+      .catch(err => {
+        return new Error(err);
```
Comment (mishelashala) [inline app/services/GoogleCalendarService.ts]: why is this necessary?

Comment (KsquarianMike) [reply app/services/GoogleCalendarService.ts]: there is an error when POST a Booking, when checking the log there is no description of the error, it would help to identify the existing error and future errors

File: .nyc_output/79309f39-2d6f-41b8-8147-14837f520fa6.json (hunk 1 above)
Comment (KsquarianMike) [reply .nyc_output/79309f39-2d6f-41b8-8147-14837f520fa6.json]: done
